### PR TITLE
LPD-28319 Invalidate WebDAV session

### PR DESCRIPTION
**Pull request by nikolettabuza**
[LPD-28319](https://liferay.atlassian.net/browse/LPD-28319)

### CODE REVIEW

**Comment by nikolettabuza** [comment]
ci:test:relevant

**Comment by nikolettabuza** [comment]
ci:test:sf

**Comment by nikolettabuza** [comment]
ci:test:relevant

**Comment by alvarosaugarlr** [comment]
ci:test:security

File: portal-impl/src/com/liferay/portal/servlet/filters/secure/BaseAuthFilter.java [hunk 1]
```diff
@@ -216,6 +220,24 @@ protected void initThreadLocals(User user) throws Exception {
...
+		throws PortalException {
+
+		User user = (User)httpSession.getAttribute(WebKeys.USER);
+
+		if (user != null) {
+			String digest = (String)httpSession.getAttribute("DIGEST");
+
+			user = UserLocalServiceUtil.getUser(user.getUserId());
+
+			if (!digest.equals(user.getDigest())) {
```
[the remark this answers is not in the document]
Comment (alvarosaugarlr) [reply portal-impl/src/com/liferay/portal/servlet/filters/secure/BaseAuthFilter.java]: Make the change in the commit - https://github.com/liferay-appsec/liferay-portal/pull/1892/commits/532a0937ace9219329b9353930d17f92643482f6

**Comment by alvarosaugarlr** [comment]
ci:test:relevant

**Comment by alvarosaugarlr** [comment]
ci:test:security

**Comment by alvarosaugarlr** [comment]
ci:test:sf

**Comment by alvarosaugarlr** [comment]
ci:test:relevant

**Comment by alvarosaugarlr** [comment]
ci:test:relevant

**Comment by alvarosaugarlr** [comment]
ci:test:sf

**Comment by nikolettabuza** [comment]
ci:test:sf

**Comment by nikolettabuza** [comment]
ci:test:relevant

**Comment by nikolettabuza** [comment]
ci:test:security

**Comment by nikolettabuza** [comment]
ci:test:sf

**Comment by nikolettabuza** [comment]
ci:test:security

**Comment by alvarosaugarlr** [comment]
For me the changes are ok and the tests too. 

The tests pass in the ci but, there is one in the same class that fail ([testHttpSessionIsInvalid](https://testray.liferay.com/#/project/34754/routines/34756/build/73484102/case-result/73539511)). @nikolettabuza Could be checked if is it related with the changes , please? 

Other doubt , when I test it manually, in the client of Fedora (Files browser), I fill the user and the password, it works but It is required another password with the message:  "Enter password for 'PortalRealm'". @stian-sigvartsen Have I made something wrong?

**Comment by nikolettabuza** [comment]
ci:test:sf

**Comment by nikolettabuza** [comment]
ci:test:relevant

**Comment by nikolettabuza** [comment]
ci:test:security

**Comment by stian-sigvartsen** [comment]
I'm working with @nikolettabuza to answer your question @alvarosaugarlr . She has a better context on this.

**Comment by stian-sigvartsen** [comment]
I'm going to close this PR because @nikolettabuza found that behaviour @alvarosaugarlr was experiencing was because of a regression described in LPD-35766. 

Once that is fixes, this PR need to be rebased and we can start another round of reviews.

**Pull request closed without merging**